src/sendMessage.js

```diff
@@ -80,10 +76,25 @@ module.exports = function(defaultFuncs, api, ctx) {
 
         send();
       });
+    } else if (msg.url || msg.uri) {
```

Comment:
Can we avoid multiple ways of doing something? I'd rather just stick to `url` as it's what we use everywhere else.